<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kitColorDialog</class>
 <widget class="QDialog" name="kitColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,QLineEdit,QRadioButton,QComboBox,QSpinBox,QTableWidget::item,QTableView::item,QCheckBox,QDateTimeEdit{
font-family:'思源黑体 CN Normal';
font-size:12pt;
}

QPushButton,QHeaderView,QGroupBox,QTabBar,QTableWidget{
font-family:'思源黑体 CN Bold';
font-size:14pt;
}

QPushButton {background-color: rgb(0, 160, 233); border-radius: 17px; color: rgb(255, 255, 255); }
QPushButton:hover {background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5,fx:0.5, fy:0.5,stop:0 rgb(0,122,232),stop:1 rgb(53,190,246) ) }
QPushButton:disabled { background-color: rgb(236, 244,255); color: rgb(180, 180, 180);}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgb(0, 160, 233);</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>33</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>思源黑体 CN Bold</family>
               <pointsize>16</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel{
        font-size:16pt;
        color:white;
        font-family:'思源黑体 CN Bold';
}</string>
             </property>
             <property name="text">
              <string>选择颜色</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>33</width>
               <height>33</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>33</width>
               <height>33</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel,QLineEdit,QRadioButton,QComboBox,QSpinBox,QTableWidget::item,QTableView::item,QCheckBox,QDateTimeEdit{
font-family:'思源黑体 CN Normal';
font-size:12pt;
}

QPushButton,QHeaderView,QGroupBox,QTabBar,QTableWidget{
font-family:'思源黑体 CN Bold';
font-size:14pt;
}

QPushButton {background-color: rgb(0, 160, 233); border-radius: 17px; color: rgb(255, 255, 255); }
QPushButton:hover {background: qradialgradient(cx:0.5, cy:0.5, radius: 0.5,fx:0.5, fy:0.5,stop:0 rgb(0,122,232),stop:1 rgb(53,190,246) ) }
QPushButton:disabled { background-color: rgb(236, 244,255); color: rgb(180, 180, 180);}</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QScrollArea" name="scrollArea">
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>896</width>
                <height>646</height>
               </rect>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <widget class="QWidget" name="widgetMain" native="true">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>896</width>
                 <height>646</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout">
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="layoutMain">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
